sdk/bare/common/drv/bsp.c

```diff
@@ -84,17 +84,13 @@ void bsp_init(void)
     led_init();
     sts_mux_init();
     dac_init();
```

Comment:
NOTE: the DAC driver *is* supported! Therefore, the comment which was removed is invalid.